packages/shared/src/hooks/analytics/useAnalyticsQueue.ts

```diff
@@ -32,6 +32,13 @@ export default function useAnalyticsQueue({
   const enabledRef = useRef(false);
   const { mutateAsync: sendEvents } = useMutation(
     async (events: AnalyticsEvent[]) => {
+      if (process.env.NODE_ENV === 'development') {
```

Comment:
I dont think we should have development-related checks on production. Maybe we can mock the backend if we want to get rid of the network errors.
@rebelchris @sshanzel @idoshamun

Reply:
Yes I agree this might be a better solution. Pretty sure there's an "echo" service we can spin up to plot all the requests it receives. It will help us debug a lot of analytics issues

Reply:
I agree that the mocked service would be a better solution long term 👍 

In terms of that service I don't know if you have something specific in mind but as `pino` is already used in api project it has a great support for logging network requests so maybe a quick endpoint for `/api/e` would be a quick win to improve debugging and just reduce the clutter of all those request errors https://github.com/pinojs/pino-http#pinohttpstdserializersreq

I am up to adding this but my concern would be where would be the best place to add `/api/e` inside daily-api to avoid interfering with production deployment. 

FYI @idoshamun If there is someone better to handle this instead of me that is of course an option.

Reply:
Probably the `daily-api` would be the right project. @rebelchris should prioritize :)

Reply:
Hmm yeah we can look at this at a later stage.
Not to dismiss it for now, but we have a couple urgent issues to deal with first.

Let's put this on the backlog and we can add it for a sprint with some more breathing room?

Reply:
@idoshamun already did it after all of the gateway changes 👌